example-config.json

```diff
@@ -30,6 +30,14 @@
         {
             "provider": "reddit",
             "config": { "subreddit": "todayilearned" }
-        }
+        },
```

Comment:
Is there a reason you added this to the example file? (I happen to read KoS plenty, just wondering since it adds runtime to the example.)

Reply:
I didn't add that. It was already there.

Reply:
Let me clarify..if you run the example against my PR it isn't processed. If you run in against the current master it would be.